tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+# This script is supposed to facilitate the preparation of a rook release by
+# changing example and documentation files to use the new release version instead the old one.
+#
+# It will usually be run on a release branch and the resulting changes will be the content of the
+# commit for the basis of the release.
+#
+#
+
+
+
+
+
+# The script needs two version tags:
+
+# FROM_TAG: the version tag of the previous/current release (e. g. v1.16.7)
+#   TO_TAG: the version tag of the new release being prepared (e. g. v1.16.8)   
+#
+# The script  replaces all occurrences of $FROM_TAG with  $TO_TAG in
+# the relevant affected files.
+
+
+# The two tags can be passed to the script as the first two positional parameters in its invocation.
+
+# So an example invocation could be:
+# set-release-ver.sh v1.16.7 v1.16.8
```

Comment:
Can this be made less verbose?

Reply:
@BlaineEXE 
> Can this be made less verbose?

It is a lot less verbose now.